pkg/configure/configure.go

```diff
@@ -182,6 +182,9 @@ func createUpdatedSecret(reconfigureRequest request, oldSecret corev1.Secret) co
 	}
 
 	for prop, value := range oldSecret.Data {
+		if prop == "ssl.key" || prop == "ssl.cert" {
+			continue
+		}
```

Comment:
I don't think we need this `oldSecret` function argument anymore. I remember introducing this to solve https://issues.redhat.com/browse/PROJQUAY-2419 but it seems like after your fixes in the config editor it works without it. Can you remove it?

Reply:
i.e. prior to your fix in the config editor it was not sending ssl.cert and ssl.key, hence this hack to inherit whatever was in the old secret.

Reply:
Removed the logic for the old secret